internal/database/mysql.go

```diff
@@ -9,7 +9,7 @@ import (
 	"log"
 	"os"
 	"strings"
-
+	"golang.org/x/crypto/bcrypt"
```

Comment:
follow import fomratting and leave a space between standard library vs module imports. Also, we are sorting alphabetically so the _ "github..." should come first in the second block.

Reply:
Understood, I have made the required changes.